packages/ui-components/src/components/DataTable/DataTable.tsx

```diff
@@ -28,7 +37,19 @@ const getColumnId = ({ id, accessor, Header }) => {
   return Header;
 };
 
-export const DataTable = ({ columns, data, className, rowLimit, total }) => {
+export const DataTable = ({
+  columns,
+  data,
+  className = '',
+  rowLimit = 0,
+  total = 0,
+}: {
+  columns: any[];
+  data: any[];
```

Comment:
I think if we know that it's an array of objects then it's good to put that. 

```suggestion
  data: object[];
```